Gordon360/Services/WellnessService.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using Gordon360.Models;
+using Gordon360.Models.ViewModels;
+using Gordon360.Repositories;
+using Gordon360.Exceptions.CustomExceptions;
+using System.Data.SqlClient;
+using Gordon360.Services.ComplexQueries;
+using System.Diagnostics;
+
+namespace Gordon360.Services
+{
+    public class WellnessService : IWellnessService
+    {
+       
+
+        public WellnessService()
+        {
+        }
+
+        /// <summary>
+        /// gets status of wellness check
+        /// </summary>
+        /// <param name="id">id</param>
+        /// <returns>boolean if found, null if not found</returns>
+
+        public IEnumerable<WellnessViewModel> GetStatus(string id)
+        {
+            var _unitOfWork = new UnitOfWork();
```

Comment:
In this instance my code is a bit different than others in the API. The main difference being that instead of making "UnitofWork" part of my constructor, I import it type and use an instance of it. This makes it so that I dont have to define my service in the "UnitofWork" file. This is something that me and Matt Felgate figured out was giving us a hard time the first time we were looking at the code. If possible, Id say it's a good idea to include your service in the "UnitofWork".